main.go

```diff
@@ -6,21 +6,25 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
 	"runtime"
+	"github.com/ghodss/yaml"
 	"strings"
 	"sync"
 
 	"github.com/mitchellh/go-homedir"
 	"github.com/xeipuuv/gojsonschema"
-	"neilpa.me/go-x/fileuri"
+
 )
 
-var (
-	version = "undefined"
```

Comment:
Can you bring back this change.

Reply:
You want it to just be a var and no constant, right?

Reply:
done (if the intention was var and no constant)